### PR TITLE
Enables Redux DevTools Chrome extension

### LINKED ISSUE #7: Redux DevTools can't find store
The store enhancer is missing some configuration:

https://github.com/zalmoxisus/redux-devtools-extension#2-use-with-redux

### PR DESCRIPTION
**Pull request by rcsole**
Redux DevTools was not being able to access the store because we needed
to set it up. This closes #7.

### CODE REVIEW

File: lib/store.js [hunk 1]
```diff
@@ -9,6 +9,15 @@ function isFunc(object) {
   return (typeof object === 'function');
 }
 
+function devTools () {
+  if (typeof window === 'object'
+      && typeof window.devToolsExtension !== 'undefined') {
```
Comment (matthias-reis) [inline lib/store.js]: Just a question: Do we want the devTools even in production mode?

Comment (rcsole) [inline lib/store.js]: Probably not, let me fix that.

Comment (dmbch) [inline lib/store.js]: Why wouldn't we, actually? I really kind of like the idea to have these kinds of debugging capabilities production, the more so as having them wouldn't even bloat our bundle...

Comment (rcsole) [inline lib/store.js]: Hmmm... I guess you're right. Worst that can happen is someone
hooks on our store with their DevTools, it's not like they access the
store itself so it should be fine.

Comment (rcsole) [inline lib/store.js]: cc @rudeworks @naoisegolden

**Comment by rcsole** [comment]
I've re-enabled it for production again, I do think it would be useful.

**Comment by dmbch** [comment]
Thank you! :+1:

**Pull request merged**